### ISSUE TITLE
How we will make decisions? 

### ISSUE THREAD

**Issue by soerenthomsen** [OceanGlidersCommunity/OG-format-user-manual#30]
Soeren: I convert this to do from Gui into an issue now.

Gui: "Until now we were a small group with a fairly similar backgrounds. How do we move forward?"

**Comment by soerenthomsen** [comment]
Based on a discussion with @castelao, I wrote something for the oxygen SOP here:
https://github.com/OceanGlidersCommunity/Oxygen_SOP/blob/main/CONTRIBUTING.md

Maybe it helps. But as you have a formal board already it might be a bit different in your case.

**Comment by justinbuck** [comment]
Where there are updates where DACs will need to regenerate file stick then the decision needs to be made by the OGDMTT. Do we need to queue additions once the format goes live and review monthly or quarterly initially. As the format becomes more established then move to annual review (like comparable observing programmers).

**Comment by castelao** [comment]
@justinbuck , I agree with you but I think that there are two different concepts there. One is the release and adoption, which makes sense for me to do in discrete time steps as you suggested. The other concept is to accept changes and improvements. This last one might be a continuous process to be included in the next release, whenever that is. Continuous or not, how will be the process to accept any addition or change? Typos and style are easy, and I'm not worried about those.

The OG Format is missing an explicit list of authors. We should start by adding that.

@soerenthomsen , on your example I would have 'adding new sensor' as a major. Different groups can use the same sensor in different ways. I think it is worth more than one reviewer, thus more perspectives. It is easy to add, but annoying to change standards.

As I mentioned before, I find CF-Conventions a good example: https://cfconventions.org/governance.html

**Comment by justinbuck** [comment]
@castelao I agree, we need to versions once OG1 goes live:
- The current implementation version for DACs to produce
-  A development version that incorporated sets of changes for approval before becoming the implementation version
Is this where the OBPS could help by pointing to the current live  implementation for DACs ?

**Comment by soerenthomsen** [comment]
@tcarval this discussion might be important for you too. Glad to have you here now :)

**Comment by glidermann** [comment]
We should propose something at this summer's EU-DM meeting...and if consensus take to OGDMTT then to OG steering for approval. Besides the side conversation in #21 is there a proposal elsewhere in this GitHub space? @vturpin

**Comment by vturpin** [comment]
This is also related to #55 and partly  #64 . Noted for June/July meeting. 

I don't see other proposal in the Github space. Shall we create a file called "decision making" or "format governance"? A bit like the 'code of conduct' ? It will describe how we will proceed to make the format evolving...

**Comment by glidermann** [comment]
Something to do in frame of OGDMTT..we need to get volunteers to create and manage/lead this. Too much for me already given my available time.
